test/Sentry.Maui.Tests/SentryMauiOptionsTests.cs

```diff
@@ -56,4 +58,35 @@ public void AttachScreenshots_Default()
         var options = new SentryMauiOptions();
         Assert.False(options.AttachScreenshot);
     }
+
+    [Fact]
+    public void BeforeCaptureScreenshot_Default()
+    {
+        // Arrange
+        Action emptyAction = () => { };
+        var options = new SentryMauiOptions();
+
+        // Act
+        //bool result = options.BeforeCaptureScreenshot == emptyAction;
+
+        // Assert 
+        //Assert.p();
```

Comment:
TODO???

Reply:
oops, i was meant to remove that actually. I thought I would implement it but I figured there is no point, so I wanted to remove it but it seems like I forgot.

thanks for noticing it